invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/ActionForm.js

```diff
@@ -76,20 +76,32 @@ export class ActionForm extends Component {
   };
 
   render() {
-    const { actionSchema } = this.props;
+    const { actionSchema, formFields, actionCancelCallback } = this.props;
     const { loading, formData, error } = this.state;
     return (
       <Formik initialValues={formData} onSubmit={this.onSubmit}>
         {(props) => (
           <SemanticForm as={Form} onSubmit={props.handleSubmit}>
-            <GenerateForm jsonSchema={actionSchema} />
+            <GenerateForm
+              jsonSchema={actionSchema}
+              formFields={formFields}
+              create
+              dropDumpOnly
```

Comment:
because sometimes you might want to display them in disabled mode in the edit form, for better UX
or you might want to drop them completely - it gives you more flexibility

Reply:
Ah, the disabled mode is already implemented and enforced if they are dump_only (I missed that), makes sense! thanks!